ansible/inventories/devnet-9/group_vars/tx_fuzz.yaml

```diff
@@ -0,0 +1,11 @@
+tx_fuzz_container_image: "{{ default_tooling_images.tx_fuzz }}"
+
+
+tx_fuzz_container_command:
```

Comment:
think we need a second blob tx-fuzz, i can add that later on - wanted to check a few things. but the secret can go in already

Reply:
will do this in a bit